apps/condo/lang/en/en.json

```diff
@@ -631,6 +631,7 @@
   "pages.condo.property.field.Address.notFound.modal.contacts.description.bot": "Telegram-bot",
   "pages.condo.property.field.Name": "Name",
   "pages.condo.property.field.Organization": "Organization",
+  "pages.condo.property.field.Inn": "Taxpayer Identification Number",
```

Comment:
Same about naming ^

Reply:
Best to just leave it as `TIN` it would not look good to render giant string in UI :-)